torch/_torch_docs.py

```diff
@@ -8928,6 +8928,50 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.kaiser_window,
+           """
+kaiser_window(window_length, beta, periodic=True, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Kaiser window function.
+
+.. math::
+    w[n] = I_0 \left( \beta \sqrt{1 - \frac{4n^2}{(M-1)^2}} \right) / I_0( \beta )
+
+with
+
+.. math::
+    -\frac{M-1}{2} \leq n \leq \frac{M-1}{2}
+
+where :math:`N` is the full window size.
+The input :attr:`window_length` is a positive integer controlling the
+returned window size. :attr:`periodic` flag determines whether the returned
+window trims off the last duplicate value from the symmetric window and is
+ready to be used as a periodic window with functions like
+:meth:`torch.stft`. Therefore, if :attr:`periodic` is true, the :math:`N` in
+above formula is in fact :math:`\text{window\_length} + 1`. Also, we always have
+``torch.kaiser_window(L, B, periodic=True)`` equal to
+``torch.kaiser_window(L + 1, B, periodic=False)[:-1])``.
+
+.. note::
+    If :attr:`window_length` :math:`=1`, the returned window contains a single value 1.
```

Comment:
newline here. This is actually interesting that this is true with periodic being True, too. What do you think about that interaction?

Reply:
All the other window functions in pytorch, and those in scipy/numpy also follow this behavior. I feel like any other value would simply not be correct, since I would expect the value to be representative of the window, and with one being located at the center of the window, I would think that for a window length of 1, that one would be what is returned.

Reply:
Sounds good.